src/controllers/EthereumBlockchainController.ts

```diff
@@ -196,7 +221,7 @@ export class EthereumBlockchainController extends AbstractBlockchainController {
 	}
 
 	async init(): Promise<void> {
-		await this.blockchainReader.init();
+		console.log('Deprecated');
```

Comment:
We no longer need to init anything in blockchain reader. This method is left since AbstractBlockchainController is defined in sdk.